evap/staff/views.py

```diff
@@ -1941,37 +1959,33 @@ def questionnaire_set_locked(request):
 
 
 @manager_required
-def degree_index(request):
-    degrees = Degree.objects.all()
-
-    DegreeFormset = modelformset_factory(
-        Degree, form=DegreeForm, formset=ModelWithImportNamesFormset, can_delete=True, extra=1
+class DegreeIndexView(SuccessMessageMixin, SaveValidFormMixin, FormsetView):
+    model = Degree
+    form_class = modelformset_factory(
```

Comment:
This is used by `FormsetView`, which is our code, right? If so, do we want to name this `formset_class` instead?

Reply:
Actually, this is used by `FormView` which is Django, but we could add this renaming functionality for `FormSetView` too. I was thinking about overriding `form_valid` and similar too so that subclasses can use `formset` as the argument name, but I wanted to wait until we decide which views we want to keep